cypress/integration/homework/classes/HomePage.js

```diff
@@ -0,0 +1,70 @@
+class HomePage {
+
+
+    clearButton = "#clearButton";
+    actionDropDown = "#selectOperationDropdown";
+    calculateButton = "#calculateButton";
+    numberField1 = "#number1Field";
+    numberField2 = "#number2Field";
+    integersOnlyCheckbox = "#integerSelect";
+
+
+    checkIfFieldsEnabled(field) {
+        cy.get(field).should('be.enabled')
+    }
+
+    enterFieldsWith(number1, number2, action) {
```

Comment:
Looks like you are using _enterFieldsWith_ method not only for entering values (I would assume that from the method title) but also for performing the action. Rethink about splitting these actions into separate methods.